Makefile.am

```diff
@@ -1,11 +1,11 @@
 SUFFIXES= .rst
 
-EXTRA_DIST = COPYING scripts/findstatic.pl swupd.bash
+EXTRA_DIST = COPYING scripts/findstatic.pl
 
 AM_CFLAGS = -fPIC -Iinclude/ -O2 -g -Wall -W -Wformat-security -D_FORTIFY_SOURCE=2 -fno-common -std=gnu99
 ACLOCAL_AMFLAGS = -I m4
 
-bin_PROGRAMS = swupd
+bin_PROGRAMS = swupd swupd.bash
```

Comment:
Was moving swupd.bash from EXTRA_DIST to bin_PROGRAMS intentional? I didn't see this mentioned in the commit description.

Reply:
Yes, Otherwise swupd.bash is only made by "make dist", so "make check" fails.
I could have fixed it another way, for example having a requirement on swupd.bash for the test which checks it, but this seemed simplest.

Reply:
Okay that makes sense now, thanks.